src/python/esp_compress.py

```diff
@@ -2,7 +2,7 @@
 import shutil
 import os, glob
 
-FIRMWARE_PACKING_ENABLED = False
+FIRMWARE_PACKING_ENABLED = True
```

Comment:
Compress will generate another file with `bin.gz` extension so this can be enabled now.